include/sounds.h

```diff
@@ -3,21 +3,28 @@
 
 // TODO: get better sound names i guess
 
+// Comments beside sounds are from
+// playing the game and observing which
+// sounds are played in which situations
+// by logging calls to function 800a7678
```

Comment:
would recommend using the decomp symbol/filename here instead of the address

Reply:
alright